### PR TITLE
Overview: connect with server and make get request to API

### PR DESCRIPTION
**Pull request by zxl3269117**

### CODE REVIEW

**Comment by zxl3269117** [comment]
@davidltruong @mpmanzo 
- Render Overview components from API data.

Of note:
I made change in api.js AND @davidltruong  `app.get(''/qa/questions'...)`. I should've pointed out when you made the pull request David. Sorry about that. 
I left comment in the file to explain the reason for the change.
No need to approve. Happy to discuss further regarding API request, param, path variable and etc. Which I found are different from what API request we'll make.

File: api.js [hunk 1]
```diff
@@ -3,7 +3,11 @@ const API_KEY = require('./config.js');
 
 let getData = (url, param, callback) => {
   let options = {
-    url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp${url}${param}`,
+    // url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp${url}${param}`,
```
Comment (zxl3269117) [inline api.js:6]: Reason for change: the get request I need to make includes a `path variable`, represented in url as `/products/:product_id`. Where `product_id` is a `path variable` not a query parameter. And, my request does not have a query parameter. So I had to change the options to work for both scenarios.

I left the original one commented out.

**Review by mpmanzo — APPROVED** [review]
Looks good

**Review by davidltruong — APPROVED** [review]
Ok no worries. Your changes looks cleaner for server/api files. Ill have to test it out to make sure it still works the way i thought it would.

**Pull request merged**